docker-compose.yml

```diff
@@ -5,32 +5,35 @@ volumes:
 services:
   tesla_ble_mqtt:
     container_name: tesla_ble_mqtt
-    image: "tesla-local-control/tesla_ble_mqtt:latest"
+    image: "iainbullock/tesla_ble_mqtt:latest"
 
-
-    # Configuration Settings have been moved to the "env" file
+    # Configuration Settings have been moved to the "stack.env" file
     env_file:
-      - env
+      - stack.env
 
+    # Needed to open the Bluetooth dongle to send commands to the car
     network_mode: host
-
-    # TODO:  Needed to open the Bluetooth dongle to send commands to the car
     privileged: true
-    # TODO: Needed to open the Bluetooth device
+
+    # TODO: Needed to open the Bluetooth device. 
+    # Reply: I think it is needed to get a container shell which is a useful diagnostic
     tty: true
 
     # TODO : Why?
+    # Reply: I think it is needed to get a container shell which is a useful diagnostic
     stdin_open: true
 
     # TODO : still valid?
-    working_dir: /share/tesla_ble_mqtt
+    # Reply: It seems to work after I remmed it out
+    # working_dir: /share/tesla_ble_mqtt
 
     volumes:
       - tesla_ble_mqtt:/share/tesla_ble_mqtt
       - /run/dbus:/run/dbus
 
-    # TODO: most likely not needed to overwrite the image's default
-    entrypoint: "/app/run.sh"
+    # Uncomment if you need a container shell when the container won't run 
+    # entrypoint: "/bin/ash"
```

Comment:
`docker exec -ti <container name or id> bash` should work, right?

Reply:
Oh when the container won't run.. got it.

Reply:
Updated as per your comments. 

As per WhatsApp: We still need to 1. update the Install.md 2. switch from stage to main branch 3. copy over the changelog from core